src/analyze/BuildSystem.re

```diff
@@ -145,27 +145,35 @@ let getCompiledBase = (root, buildSystem) =>
     | BsbNative(_, Native) => root /+ "lib" /+ "bs" /+ "native"
     | BsbNative(_, Bytecode) => root /+ "lib" /+ "bs" /+ "bytecode"
     | Dune(Esy(esyVersion)) => root /+ getEsyCompiledBase(esyVersion)
+    | Dune(Opam) => root /+ "_build"
```

Comment:
what about checking if `DUNE_BUILD_DIR` is set and falling back to `_build` if not?

Reply:
:+1: can enhance

Reply:
On second thought, your suggestion assumes that the editor would be launched with these variables in its env, as in e.g. `opam exec -- code .`, right? It looks like I may have my answer from a convo in https://github.com/jaredly/reason-language-server/pull/174/files#r231520320 but wanted to make sure the same idea applies here.

Relatedly, is there some reason I'm unaware of to parse `Unix.environment` instead of just using `Sys.getenv_opt`?